src/Components/v2/Lightbox/Lightbox.tsx

```diff
@@ -278,42 +279,31 @@ export class Lightbox extends React.Component<LightboxProps, LightboxState> {
   }
 
   render() {
-    const { children, enabled, isDefault, imageAlt } = this.props
+    const { children, enabled, isDefault, imageAlt, src } = this.props
 
     // Only render client-side
     if (!this.state.element) {
       return children
     }
 
-    const modifiedChildren = React.Children.map(
-      children,
-      (child: React.ReactElement<any>) => {
-        const {
-          props: { src },
-        } = child
-        return (
-          enabled && (
-            <Flex
-              justifyContent="center"
-              alignItems="center"
-              height="60vh"
-              onClick={this.show.bind(this)}
-            >
-              <StyledImage
-                src={src}
-                alt={imageAlt}
-                data-type="artwork-image"
-                data-is-default={isDefault}
-              />
-            </Flex>
-          )
-        )
-      }
-    )
```

Comment:
Mapping over children to derive props is ok, and is a a common approach when writing lib-like code (such as `Lightbox`. 

The way to enforce that only one child is present is via the `React.Children.only`: https://reactjs.org/docs/react-api.html#reactchildrenonly. 

That way this all of this stuff can be self-contained, and the api remain configuration-less (vs moving the container and so on _outside_ of the component.